src/gluonts/torch/__init__.py

```diff
@@ -11,12 +11,25 @@
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
 
-# !!! DO NOT MODIFY !!! (pkgutil-style namespace package)
+__all__ = [
+    "PyTorchLightningEstimator",
+    "PyTorchPredictor",
+    "DeepNPTSEstimator",
+    "DeepAREstimator",
+    "MQF2MultiHorizonEstimator",
+    "SimpleFeedForwardEstimator",
+    "TemporalFusionTransformerEstimator",
+]
 
 
-from pkgutil import extend_path
+from .model.estimator import PyTorchLightningEstimator
+from .model.predictor import PyTorchPredictor
 
-__path__ = extend_path(__path__, __name__)  # type: ignore
+from .model.deep_npts import DeepNPTSEstimator
+from .model.deepar import DeepAREstimator
+from .model.mqf2 import MQF2MultiHorizonEstimator
```

Comment:
This line adds a dependency on `cpflows` which I’m not sure we want to require at this level. See https://github.com/awslabs/gluonts/blob/dev/requirements/requirements-extras-cpflows.txt